tools/toolchains/__init__.py

```diff
@@ -551,6 +550,12 @@ def get_labels(self):
                 self.labels['TARGET'].append("RELEASE")
         return self.labels
 
+    def _get_toolchain_labels(self):
+        toolchain_labels = [c.__name__ for c in getmro(self.__class__)]
+        toolchain_labels.remove('mbedToolchain')
+        toolchain_labels.remove('object')
```

Comment:
Why is 'object' now also being removed?

Reply:
Because it was never supposed to be there in the first place and adds no information.